EIPS/eip-969.md

```diff
@@ -0,0 +1,167 @@
+---
+eip: 969 
+title: Modifications to ethash to invalidate dedicated hardware implementations 
```

Comment:
I suggest "Modifications to ethash to invalidate _existing_ dedicated hardware implementations"